sandBox/FirstSprint/TaskLoops.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Concurrent;
+using System.Linq;
+using System.Text;
+
+namespace FirstSprint
+{
+    class TaskLoops
+    {
+        private string[] GetRandomStringArray()
```

Comment:
Ideally such things will be split and stored in some kind of data randomizer class that will be widely used after some time and you will have separate methods - one to get random number, second to get random string of defined length. Thus you will be able to pass result from first to second and get random length random string.